rotkehlchen/exchanges/bitfinex.py

```diff
@@ -802,6 +795,60 @@ def _process_unsuccessful_response(
         raise AssertionError(f'Unexpected {self.name} response_case: {case}')
 
     def first_connection(self) -> None:
+        """Request:
+        - currencies list: tickers of the supported assets.
+        - pairs list: pairs available for trading.
+        - currency map: a map between the ticker used by Bitfinex and the
+        standard one (outside Bitfinex).
+
+        These data are stored in properties along with `pair_bfx_symbols_map`,
+        a dict that maps a pair between the tickers of the base and quote assets.
+        """
+        if self.first_connection_made:
+            return
+
+        currencies_response = self._query_currencies()
+        if currencies_response.success is False:
+            raise RemoteError(
+                f'{self.name} failed to request currencies list. '
```

Comment:
Why self.name here? It's always bitfinex, isn't it? Don't use a variable if it's not needed. Here and below.

Reply:
Sure. I've did it for consistency. I've been using `{self.name}` on each exchange I've implemented as it allows me to re-use logs and be consistent with the messages. But I understand as well your point of view in particular on these ones (very simple).

Reply:
Yes makes sense. My comment was a bit nitpicky. Apologies for that.